webshare/share-securecontext.http.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I was unable to test this file locally; it's supposed to run on a non-secure context but I am testing on localhost because I can't get OpenSSL working. So... does the test runner automatically open "http.html" pages on a non-secure origin? Otherwise this test isn't going to work.